### ISSUE TITLE
[Bug] = temporal-delivery-app initialization: `Error: ENOENT: no such file or directory tsconfig.json`

### ISSUE THREAD

**Issue by cliffordfajardo** [temporalio/samples-typescript#251]
### What are you really trying to do?
I am attempting to follow the Temporal NodeJS delivery app guide from this blogpost: [Building Reliable Distributed Systems in Node.js](https://temporal.io/blog/building-reliable-distributed-systems-in-node)

### Describe the bug
After running the first command specified in the tutorial, I am getting an error:

```sh
  ~/Repos/temporal-playgrounds npx @temporalio/create@latest --sample food-delivery
Need to install the following packages:
  @temporalio/create@latest
Ok to proceed? (y)
✔ What is your project named? … temporal-delivery-app-example
Creating a new Temporal project in /Users/cfajardo/Repos/temporal-playgrounds/temporal-delivery-app-example/

Downloading files for sample food-delivery. This might take a moment.

Installing packages. This might take a couple of minutes.

Aborting installation.
Unexpected error. Please report it as a bug:
[Error: ENOENT: no such file or directory, open '/Users/cfajardo/Repos/temporal-playgrounds/temporal-delivery-app-example/tsconfig.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'open',
  path: '/Users/cfajardo/Repos/temporal-playgrounds/temporal-delivery-app-example/tsconfig.json'
}
```

### Minimal Reproduction
1.Attempt to initialize the delivery app example using the following command used in the blog post:

```sh
npx @temporalio/create@latest --sample food-delivery
```

### Environment/Versions
- OS: Mac Ventura 13.2
- NodeJS version 17.9.1
- Temporal v1.6.0 typescript
- Not using Docker or Kubernetes or building Temporal from source?

## References
- [Building Reliable Distributed Systems in Node.js](https://temporal.io/blog/building-reliable-distributed-systems-in-node)

**Comment by lorensr** [comment]
Hey, thanks for reporting and including your Node version! I had only tested it in Node 16, but now I'm reproducing your error in 17 and 18. Looking into it now. In the meantime, Node 16 should work.

**Comment by jhubbardsf** [comment]
@lorensr While this is fixed in the sdk-typescript main repository it doesn't look like the @temporal/create package has been published in a long time. Last version (1.6.0) was published two months ago. I came across this same bug trying to create the monorepos sample.

**Comment by lorensr** [comment]
Sorry about that! For now, you can clone this repo and cd into food-delivery

**Comment by jhubbardsf** [comment]
I ended up cloning the sdk-typescript package and running the cli.js file from main because I was curious about the bug. Which led me to another thought. It appears the fix for it is to just ignore the error if tsconfig.ts doesn't exist. But, for example in the monorepo-folders sample, it would need to check that packages folder and then do the node version replacing in each of them for that part of the code to work correctly (I had just skimmed it).

If the intent is to change the `tsconfig/node16/tsconfig.json` lines to be the version of node currently running on the system.

**Comment by jhubbardsf** [comment]
@lorensr I opened a [PR](https://github.com/temporalio/sdk-typescript/pull/1089) in the sdk-typescript repo that I think will fix the underlying issues and make the try/catch unnecessary.

**Comment by mjameswh** [comment]
Fixed in temporalio/sdk-typescript#1089. Thanks @jhubbardsf.
